tests/test_signed_payload.py

```diff
@@ -3,74 +3,120 @@
 import json
 import base64
 
-
 class TestApi(TestCase):
-    def print_payload_signature(self):
+    def sign_message(self, message):
         pk = keys.PrivateKey(b'\x01' * 32)
+        signature = pk.sign_msg(message)
+        signature_bytes =signature.to_bytes()
```

Comment:
Space after `=`.
Could you run `./lint` and check for new style warnings for files you've modified?

Reply:
checked